blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -65,6 +65,7 @@ export class RunRecord extends Record({
     state: null,
     type: null,
     commitId: null,
+    actions: null,
 }) {
     isQueued() {
```

Comment:
IMO ```getActions``` should return the object array and not only the ```_class``` 

If you want such a method then it should be something like ```getActionClasses```